VDUBStep/Classes/VDUBStore.h

```diff
@@ -13,9 +13,8 @@
 + (VDUBStore *)sharedInstance;
 
 + (NSManagedObjectContext *)mainContext;
++ (NSManagedObjectContext *)backgroundContext;
 
 - (void)setupCoreDataStackWithURL:(NSURL *)storeURL;
 
-- (NSManagedObjectContext *)mainContext;
```

Comment:
Kendall, I'm curious what you think about my preferring class-level methods for getting at context. 

    [VDUBStore mainContext]

vs.

    [[VDUBStore sharedInstance] mainContext]

I don't think we need both class- and instance-level methods and can't think of a circumstance where the instance-level method is preferable. I like the conciseness of the former.